lms/djangoapps/grades/new/course_grade.py

```diff
@@ -40,6 +40,7 @@ def subsection_grade_totals_by_format(self):
                     graded_total = subsection_grade.graded_total
                     if graded_total.possible > 0:
                         subsections_by_format[subsection_grade.format].append(graded_total)
+        log.warning("calculated subsections_by_format property: {0}".format(unicode(subsections_by_format)))
```

Comment:
Could we include the student in that statement as well? Otherwise we won't be able to differentiate log statements for different users in the same course.